airsonic-main/src/main/java/org/airsonic/player/controller/CoverArtController.java

```diff
@@ -236,7 +241,16 @@ private void sendUnscaled(CoverArtRequest coverArtRequest, HttpServletResponse r
 
     private File getCachedImage(CoverArtRequest request, int size) throws IOException {
         String hash = DigestUtils.md5Hex(request.getKey());
-        String encoding = request.getCoverArt() != null ? "jpeg" : "png";
+        String encoding = null;
+        ImageWriteParam params = null;
+        if (request.getCoverArt() != null) {
+            encoding = "jpeg";
+            params = new JPEGImageWriteParam(null);
+            params.setCompressionMode(ImageWriteParam.MODE_EXPLICIT);
+            params.setCompressionQuality(0.95f);
```

Comment:
I would think quality of 90 should be more than sufficient for pretty much everything.  We probably don't want to unnecessarily blow up the image size too dramatically.